src/components/movieForm/index.tsx

```diff
@@ -12,12 +12,42 @@ import useStyles from './styles';
 import { IMovie, OptionOnSelectData } from '../../shared/types';
 import { TriangleDown12Filled } from '@fluentui/react-icons';
 import { genres } from '../../shared/constants';
+import { Controller, SubmitHandler, useForm } from 'react-hook-form';
+import { yupResolver } from '@hookform/resolvers/yup';
+import * as yup from 'yup';
+
+const schema = yup
+  .object({
+    title: yup.string().required('Title is required'),
+    vote_average: yup
+      .string()
+      .required('Average vote is required')
```

Comment:
<img width="717" alt="image" src="https://user-images.githubusercontent.com/81073/235852826-c9028e0d-7c74-4c98-b3da-4e59ed6b0875.png">
Looks like a typo, you have different names for the same field in the label and the error.